packages/devtools_app/lib/src/shared/survey.dart

```diff
@@ -40,8 +40,8 @@ class SurveyService {
     if (!server.isDevToolsServerAvailable) return null;
 
     _cachedSurvey ??= await _fetchSurveyContent();
-    if (_cachedSurvey != null) {
-      await server.setActiveSurvey(_cachedSurvey!.id);
+    if (_cachedSurvey != null && _cachedSurvey!.id != null) {
```

Comment:
why not `if (_cachedSurvey?.id != null)` instead?

Reply:
done